src/lib.rs

```diff
@@ -165,7 +166,7 @@ impl<K: Radix + Ord + Copy, V> RadixHeapMap<K, V> {
     /// The current top value. All keys pushed onto the heap must be smaller than this value.
     #[inline]
     pub fn top(&self) -> Option<K> {
-        self.top
+        self.top.clone()
```

Comment:
I could instead change this to return a reference to the top if you want to break API